great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -247,7 +241,7 @@ def _init_expectation_configuration_builders(
 
         return expectation_configuration_builders
 
-    def profile(
+    def run_profiler(
```

Comment:
I may be missing some context here, but I understood `run_profiler()` to be a convenience method at the `DataContext` level (ie. `DataContext.run_profiler()`).  Do we want to have the same method at the `Profiler`-level?  

 I'm concerned that it would be "too" different in behavior than other profilers that may inherit from base `Profiler`-class (under `great_expectations/profile/`), but I maybe over-thinking this.

Reply:
@Shinnnyshinshin Good point -- changing to `run()` so that we can use `run_profiler()` at the `DataContext` level, just like you wrote -- thank you for the good catch!